packages/gatsby-source-medium/src/gatsby-node.js

```diff
@@ -32,15 +32,29 @@ exports.sourceNodes = async ({ boundActionCreators }, { username }) => {
     const result = await fetch(username)
     const json = JSON.parse(strip(result.data))
 
-    const { posts } = json.payload
-    const collectionKeys = Object.keys(json.payload.references.Collection)
-    const userKeys = Object.keys(json.payload.references.User)
-
-    const importableResources = [
-      userKeys.map(key => json.payload.references.User[key]),
-      posts,
-      collectionKeys.map(key => json.payload.references.Collection[key]),
-    ]
+    let importableResources = []
```

Comment:
Is there a reason why you first declare `importableResources` and then change the values? The way it was before I felt it was easier to grasp.

Otherwise it's looking great!

Reply:
I liked the way it was done too, but I struggled to find a good way to conditionally insert the values while also keeping `importableResources` in the correct scope.  I'm sure there's a better way to do it (there always is), but I think this way is good enough for now at least to fix the issue with not being able to fetch users.  Doing it this way at least I know it's going to work every time.

If I get time in the next few weeks I'd like to revisit it to find those better ways, and also create some unit tests since there aren't any yet.